src/main/java/core/basesyntax/fruitshop/file/reader/TaskReaderImpl.java

```diff
@@ -0,0 +1,26 @@
+package core.basesyntax.fruitshop.file.reader;
+
+import java.io.BufferedReader;
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.List;
+
+public class TaskReaderImpl implements TaskReader {
+
```

Comment:
check all your classes and remove redundant empty lines after class declarations